src/components/Form.js

```diff
@@ -3,14 +3,15 @@ import injectSheet from 'react-jss'
 
 import H2 from 'components/H2'
 
-import { paper, shadow } from 'constants/styles/colors'
+import { paper, shadow, black, yellow } from 'constants/styles/colors'
+import { robotoBold } from '../constants/styles/fonts'
```

Comment:
Import this from `constants/styles/fonts` rather than this autogenerated relative path